src/types/utils.cpp

```diff
@@ -3,7 +3,10 @@
 
 #include "precomp.h"
 #include "inc/utils.hpp"
+#include "inc/convert.hpp"

```

Comment:
So this is awkward. I cannot include "convert.hpp" here because it will cause TerminalConnection having a link error as seen here https://github.com/microsoft/terminal/pull/7578/checks

 I'm thinking maybe create a separate `stringConvert.hpp` would fix this?

Reply:
Would it be easier to use a wstring_view with `L"name"sv`? That way we get to avoid conversion.

Reply:
Oh I thought using string would save us a few bytes. So I'll just use wstring_view and call it a day.